CoreSecurity/test/unit/src/au/gov/asd/tac/constellation/security/password/PasswordUtilitiesNGTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2010-2024 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.security.password;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.NoSuchElementException;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import org.openide.util.Lookup;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.fail;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author Quasar985
+ */
+public class PasswordUtilitiesNGTest {
+
+    /**
+     * Test of getIV method, of class PasswordUtilities.
+     */
+    @Test
+    public void testGetIV() {
+        System.out.println("getIV");
+        PasswordUtilities.reset();
+
+        final byte[] firstResult = PasswordUtilities.getIV();
+        assertNotNull(firstResult);
+        final byte[] secondResult = PasswordUtilities.getIV();
+        assertEquals(firstResult, secondResult);
+    }
+
+    /**
+     * Test of getIV method, of class PasswordUtilities.
+     */
+    @Test
+    public void testGetIVException() {
+        System.out.println("getIV Exception");
+        PasswordUtilities.reset();
+
+        // Exception can only be tested first
+        final Lookup mockLookup = mock(Lookup.class);
+        when(mockLookup.lookup(PasswordSecret.class)).thenReturn(null);
+        try (final MockedStatic<Lookup> mockLookupStatic = Mockito.mockStatic(Lookup.class)) {
+            // Setup static mock
+            mockLookupStatic.when(Lookup::getDefault).thenReturn(mockLookup);
+
+            PasswordUtilities.getIV();
+            // If exception was not thrown, fail
+            fail("NoSuchElementException should have been thrown");
+        } catch (NoSuchElementException e) {
```

Comment:
instead of doing this, you can add the `expectedExceptions` parameter to the Test annotation, which is a cleaner way of doing the same thing:
`@Test(expectedExceptions = NoSuchElementException.class)`